idd/Energy+.idd.in

```diff
@@ -788,22 +788,36 @@ ZoneAirContaminantBalance,
-       \memo balance calculation will attempt to enforce conservation of mass for each zone.
-       \memo If mixing is "No" and infiltration is "None", then the zone air mass flow
-       \memo calculation defaults to assume self-balanced simple flow mixing and infiltration objects.
+       \memo Enforces the zone air mass flow balance by either adjusting zone mixing object flow only, 
+       \memo adjusting zone total return flow only, zone mixing and the zone total return flows, 
+       \memo or adjusting the zone total return and zone mixing object flows. Zone infiltration flow air 
+       \memo flow is increased or decreased depending user selection in the infiltration treatment method.
+       \memo If either of zone mixing or zone return flow adjusting methods or infiltration is active, 
+       \memo then the zone air mass flow balance calculation will attempt to enforce conservation of 
+       \memo mass for each zone. If flow balancing method is "None" and infiltration is "None", then the
+       \memo zone air mass flow calculation defaults to assume self-balanced simple flow mixing and 
+       \memo infiltration objects.
        \unique-object
        \min-fields 3
-  A1,  \field Adjust Zone Mixing For Zone Air Mass Flow Balance
-       \note If Yes, Zone mixing object flow rates are adjusted to balance the zone air mass flow
-       \note and additional infiltration air flow may be added if required in order to balance the
-       \note zone air mass flow.
-       \type choice
-       \key Yes
-       \key No
-       \default No
+  A1,  \field Adjust Zone Mixing and Return For Air Mass Flow Balance
+       \note If "AdjustMixingOnly", zone mixing object flow rates are adjusted to balance the zone air mass
+       \note flow and zone infiltration air flow may be increased or decreased if required in order to balance
+       \note the zone air mass flow. If "AdjustReturnOnly", zone total return flow rate is adjusted to balance 
+       \note the zone air mass flow and zone infiltration air flow may be increased or decreased if required 
+       \note in order to balance the zone air mass flow. If "AdjustMixingThenReturn", first the zone mixing 
+       \note objects flow rates are adjusted to balance the zone air flow, second zone total return flow rate 
+       \note is adjusted and zone infiltration air flow may be increased or decreased if required in order to
+       \note balance the zone air mass flow. If "AdjustReturnThenMixing", first zone total return flow rate is 
+	   \note adjusted to balance the zone air flow, second the zone mixing object flow rates are adjusted and 
+	   \note infiltration air flow may be increased or decreased if required in order to balance the zone
+	   \note air mass flow.	   
+       \type choice
+       \key AdjustMixingOnly
+       \key AdjustReturnOnly
+       \key AdjustMixingThenReturn
+       \key AdjustReturnThenMixing
+       \key None
+       \default None
```

Comment:
This is modified ZoneAirMassFlowConservation object that allows to a different options of enforcing zone air flow balance. Please check the PR for details of comments. We have added three new options now.

Reply:
Tab characters that I will fix right before merging.